src/webgpu/api/validation/createRenderPipeline.spec.ts

```diff
@@ -187,3 +238,34 @@ g.test('sample_count_must_be_valid')
 
     t.doCreateRenderPipelineTest(isAsync, _success, descriptor);
   });
+
+g.test('pipeline_fragment_output_type_must_be_compatible_with_target_color_state_format')
+  .params(u =>
+    u
+      .combine('isAsync', [false, true])
+      .combine('format', kRenderableColorTextureFormats)
+      .beginSubcases()
+      .combine('sampleType', [
+        'float',
+        'uint',
+        'sint',
+        'unfilterable-float',
+      ] as GPUTextureSampleType[])
```

Comment:
```suggestion
      ] as const)
```
should work, and is typo-proof